xbmc/cores/dvdplayer/DVDInputStreams/DVDInputStreamHTSP.h

```diff
@@ -45,8 +45,6 @@ class CDVDInputStreamHTSP
   bool            NextChannel(bool preview = false);
   bool            PrevChannel(bool preview = false);
   bool            SelectChannelByNumber(unsigned int channel);
-  bool            SelectChannel(const PVR::CPVRChannel &channel) { return false; }
-  static bool     GetSelectedChannel(PVR::CPVRChannel *channel) {return false; }
   bool            UpdateItem(CFileItem& item);
 
   bool            CanRecord()         { return false; }
```

Comment:
we should really remove this at some point soon. iirc nobody but @elupus was using htsp outside pvr, and it's terribly outdated. not really related to this PR though ;-)